### PR TITLE
Create pmd.yml

### PR DESCRIPTION
**Pull request by JediSebas**

### CODE REVIEW

File: .github/workflows/pmd.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: PMD
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '31 16 * * 4'
```
Comment (yarpo) [inline .github/workflows/pmd.yml:9]: are you sure you want a cron job?

**Comment by yarpo** [comment]
OK, please see logs:
https://github.com/JediSebas/Shapes/actions/runs/4398911729/jobs/7703086261#step:4:21

and make sure all of them are addressed.

**Review by yarpo — APPROVED** [review]
There are some warnings in logs:
https://github.com/JediSebas/Shapes/actions/runs/4398911729/jobs/7703086261#step:4:21

please look at them and see if there is anything to improve (maybe there isn't or you should re-configure PMD so that they will not be logged).

File: .github/workflows/pmd.yml (hunk 1 above)
Comment (JediSebas) [reply .github/workflows/pmd.yml:9]: I didn't know what is, I need to read more, my bad

**Pull request merged**